libmscore/rest.cpp

```diff
@@ -196,6 +196,9 @@ QRectF Rest::drag(EditData& ed)
 
 bool Rest::acceptDrop(EditData& data) const
       {
+      if (measure()->isMMRest()) { // avoid crash
+            return false;
+            }
```

Comment:
Is this definitely right though? Seems you can safely add some elements to multi-measure rests?

Reply:
You could use the artifact from the corresponding 3.x PR to test this

Reply:
Question: is this about dropping an MM Rest (onto something), or about dropping (something) onto an MM rest, or both?